workflow-templates/zephyr/README.md

```diff
@@ -2,23 +2,25 @@
 
 <!-- vim-markdown-toc GFM -->
 
-- [Description](#description)
-- [Dependencies](#dependencies)
-  - [GitHub action secrets](#github-action-secrets)
-- [How to use](#how-to-use)
-  - [Release process](#release-process)
-  - [Pull requests](#pull-requests)
-- [How to configure build](#how-to-configure-build)
-  - [Packaging build artefacts](#packaging-build-artefacts)
-  - [Adding extra text to the Release notes](#adding-extra-text-to-the-release-notes)
-- [Twister workflow](#twister-workflow)
-  - [Artefacts and reports](#artefacts-and-reports)
-- [CodeChecker workflow](#codechecker-workflow)
-  - [Diff analysis results](#diff-analysis-results)
-- [A short note about Make](#a-short-note-about-make)
-- [Required GitHub action secrets](#required-github-action-secrets)
-  - [General secrets](#general-secrets)
-  - [CodeChecker specific secrets](#codechecker-specific-secrets)
+- [Zephyr](#zephyr)
```

Comment:
For some reason this line was added (maybe the TOC plugin in the VSCode does that?) Please remove it.

Reply:
I have added the `<!-- omit in toc -->` comment to omit it from the table